cmake/CMakeLists.txt

```diff
@@ -85,7 +85,7 @@ set(protobuf_BUILD_TESTS OFF CACHE BOOL "Build protobuf tests" FORCE)
 set(NSYNC_ENABLE_TESTS OFF CACHE BOOL "Build protobuf tests" FORCE)
 set(ONNX_ML 1)
 
-if(onnxruntime_USE_OPENMP)
+if(onnxruntime_USE_OPENMP AND WIN32)
```

Comment:
On Windows, it should be possible to only use Intel's OMP library (excluding VCOMP) when OMP and MKLML are both enabled. by using /link /nodefaultlib:vcomp libiomp5md.lib
see https://software.intel.com/en-us/cpp-compiler-developer-guide-and-reference-using-the-openmp-libraries#C28B029B-7E9F-46D7-8C84-38B0683236B9